pyroomacoustics/tests/test_random_ism.py

```diff
@@ -0,0 +1,176 @@
+
+"""
+Created on Tue Feb  8 17:50:45 2022
+
+@author: od0014
+
+Script to test removal of sweeping echoes with randomized image method
```

Comment:
The current test file is not suitable for automatic testing. Functions with name prefixed by `test_` are automatically run at test time by the CI system. They should not take any argument. The testing inside is typically done by using `assert` on some condition that is `True` if the code being tested is working as expected and `False` otherwise.